rfcs/rfc-009-staking.md

```diff
@@ -0,0 +1,59 @@
+
+| Deposit Size | Annual Validator Interest |
+|--------------|---------------------------|
+| 2.5M ETH     | 10.12%                    |
+| 10M ETH      | 5.00%                     |
+| 20M ETH      | 3.52%                     |
+| 40M ETH      | 2.48%                     |
+
+This has the nice property of potentially incentivizing participation around a target deposit size. Incentivisation of specific participation rates more directly (rather than deposit size) may something also worth exploring.
+
+The specifics of the Solana validator reward scheme are to be worked out in parallel with a design for transaction fee assignment as well as our storage mining reward scheme.
+
+### Slashing rules
+Unlike proof-of-work where off-chain capital expenses are already deployed at the time of block construction/voting, proof-of-stake systems require capital-at-risk to prevent a logical/optimal strategy of multiple chain voting. We intend to implement 'slashing' rules which, if broken, result some amount of the offending validator's deposited stake to be removed from circulation. Given the ordering properties of the PoH data structure, we believe we can simplify our slashing rules to the level of a voting lockout time assigned per vote.  
+
+I.e. Each vote has an associated lockout time (PoH duration) that represents a duration by any additional vote from that validator must be in a PoH that contains the original vote, or a portion of that validator's stake is slashable. This duration time is a function of the initial vote PoH count and all additional vote PoH counts.  It will likely take the form:
+
+Lockout<sub>i</sub>(PoH<sub>i</sub>, PoH<sub>j</sub>) = PoH<sub>j</sub> + K * exp((PoH<sub>j</sub> - PoH<sub>i</sub>) / K)
+
+Where PoH<sub>i</sub> is the height of the vote that the lockout is to be applied to and PoH<sub>j</sub> is the height of the current vote on the same branch. If the validator submits a vote on a different PoH branch on any PoH<sub>k</sub> where k > j > i and PoH<sub>k</sub> < Lockout(PoH<sub>i</sub>, PoH<sub>j</sub>), then a portion of that validator's stake is at risk of being slashed.
+
+The exponential nature of the lockout time protects against the ability for adversarial nodes, that may have the ability to generate faster (against clock time) PoH streams, from creating partitions and censoring nodes. Due to the highly optimized hardward properties of the `sha256` hash function, a faster PoH stream from a validator isn't likely to be significant and their PoH stream will increase in difference only linearly. Additionally, leader rotation times reduces the global speedup available to a malicious node to the fraction speed-up they've achieved multiplied by the % of stake they have in the total deposit (i.e. the % of time they are elected leader).
```

Comment:
This is very dense and language like "isn't likely" comes off as dismissive of an attack vector that otherwise could be acknowledged with more precision. I'd think this could be moved to a separate section that describes independent attack vectors and proposed solutions.  And "Leader rotation speedup" looks more like "risk mitigation" than a solution. If so, it'd be great to see that documented here.